src/__tests__/utils/device.test.json

```diff
@@ -0,0 +1,3842 @@
+[
+    {
```

Comment:
Could be good to add a comment here explaining where these came from (could just add a key like `"//": "your comment here"`)

Reply:
That's a beautifully sneaky way to put comments in JSON 🙌